clap_complete/examples/test-dynamic.rs

```diff
@@ -0,0 +1,168 @@
+use clap::{FromArgMatches, Subcommand};
+
+fn main() {
```

Comment:
I'm tempted to say we just put both types of  completions in the same "example".  Less noise in the source, fewer files to keep in sync.

Thoughts?

Reply:
well we'd need to compile it into two different binaries? Not sure what the best way to do so would be... Can examples depend on each other?

Reply:
What I'm suggesting is only having a single binary.  We could adjust the code in `common` to allow separate registrations for static vs dynamic completions.